src/Yoti.Auth/Yoti.Auth.csproj

```diff
@@ -16,10 +16,10 @@
     <PackageIcon>Yoti.png</PackageIcon>
     <RepositoryUrl>https://github.com/getyoti/yoti-dotnet-sdk</RepositoryUrl>
     <PackageTags>yoti 2FA multifactor authentication verification identity login register verify 2Factor sdk sandbox documents docs KYC AML</PackageTags>
-    <GeneratePackageOnBuild>false</GeneratePackageOnBuild>
+    <GeneratePackageOnBuild>False</GeneratePackageOnBuild>
     <LangVersion>latest</LangVersion>
     <DisableImplicitNuGetFallbackFolder>true</DisableImplicitNuGetFallbackFolder>
-    <Version>3.11.0</Version>
+    <Version>3.12.1-beta</Version>
```

Comment:
Would prefer not to have this as the version in development :+1: we should update when we can

Reply:
100% agreed, I only left it like this so that in both our absences (holidays) there was no confusion about the exact codebase the package I delivered to the slack channel was generated from. Can ammend in next PR. Thanks Ed.